diagnostic.md

```diff
@@ -53,9 +68,11 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
+1. .big
+2. #contentPane
+3. .important, .red
+4. div p
```

Comment:
The space acts as a descendent relationship. In this case, we are a parent/child relationship so we need `div > p` to represent that. Big difference between descendent and child.